pkgs/development/ocaml-modules/mparser-pcre/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub, ocamlPackages, buildDunePackage, ocaml }:
```

`ocamlPackages.callPackage` is scoped:

```suggestion
{ lib, fetchFromGitHub, buildDunePackage, ocaml_pcre, mparser }:
```